app/controllers/flood_risk_engine/start_forms_controller.rb

```diff
@@ -17,7 +17,7 @@ def transient_registration_attributes
     end
 
     def find_or_initialize_transient_registration(token)
-      @transient_registration = NewRegistration.where(token: token).first ||
+      @transient_registration = NewRegistration.where(token:).first ||
```

Comment:
Not sure I like this new Rubocop recommendation, might be worth discussing whether to disable it globally.

Reply:
I just think the old format was easier to scan. Since you've done the work here let's leave it for now, but let's review before the other projects are migrated to Ruby v3+.